terraform/environments/electronic-monitoring-data/modules/lambdas/variables.tf

```diff
@@ -21,17 +29,22 @@ variable "role_name" {
 variable "handler" {
   description = "The function entrypoint in your code."
   type        = string
+  nullable = true
+  default = null
 }
 
 variable "layers" {
   description = "List of Lambda Layer Version ARNs to attach to your Lambda Function."
   type        = list(string)
   nullable    = true
+  default = null
```

Comment:
should this have a default nullable of true?

Reply:
yeah - if we containerise we dont need a layer